lib/crawlers/company_crawler/company_crawler.rb

```diff
@@ -0,0 +1,80 @@
+require_relative "../../priority_spidr/priority_spidr"
+require_relative "../base_crawler"
+require "cgi"
+require "pathname"
+require "csv"
+require "uri"
+
+module Crawlers
+  class CompanyCrawler < BaseCrawler
+    ROOT = Pathname.new(__FILE__).parent.parent.parent.parent
```

Comment:
Can this be tidied / done another way?

Reply:
Yeah, I'm moving some stuff around so we can use `rails runner {file}` instead of `ruby {file}` so that the file gets access to `Rails.root` instead of doing this.